crates/solver/src/settlement_submission/submitter/common.rs

```diff
@@ -27,7 +26,7 @@ impl PrivateNetwork {
         &self,
         tx: TransactionBuilder<Web3Transport>,
     ) -> Result<TransactionHandle> {
-        let (raw_signed_transaction, tx_hash) = match tx.build().now_or_never().unwrap().unwrap() {
+        let (raw_signed_transaction, tx_hash) = match tx.build().await.unwrap() {
```

Comment:
Don't unwrap. This used to not fail because we required an offline account but with KMS it can fail.